glusterd2/commands/volumes/volume-start.go

```diff
@@ -24,6 +24,10 @@ func startAllBricks(c transaction.TxnCtx) error {
 		return err
 	}
 
+	err := generateBrickVolfiles(c)
```

Comment:
Why do we need this change when we already generated brick volfile as a step function

Reply:
looks duplicate. I will relook into this